test/elm/comparison/test.coffee

```diff
@@ -108,13 +108,13 @@ describe 'NotEqual', ->
     @aLtB_Quantity_diff.exec(@ctx).should.be.true()
 
   it 'should be null for 5 Cel != 4 m', ->
-    should(@aGtB_Quantity_incompatible.exec(@ctx)).be.null()
+    should(() => @aGtB_Quantity_incompatible.exec(@ctx)).throw("Incompatible Types 'm' and 'Cel'")
```

Comment:
The title of this test does not match what the test is doing. This is an issue with all remaining test changes in this file.

Reply:
Good catch, I can update these test descriptions.